### DISCUSSION TITLE
Stopping Frigate Container does not update mqtt frigate/available

### DISCUSSION THREAD

**Discussion by pdobrien3** [blakeblackshear/frigate#10834] [General Support]
### Describe the problem you are having

https://docs.frigate.video/integrations/mqtt/#frigateavailable explains that Possible message are: "online": published when Frigate is running (on startup) "offline": published right before Frigate stops.  This topic never goes to offline when stopping the container 

### Version

0.13.2-6476F8A

### Frigate config file

```yaml
N/A
```

### Relevant log output

```shell
None
```

### FFprobe output from your camera

```shell
N/A
```

### Operating system

Debian

### Install method

Docker Compose

### Coral version

USB

### Network connection

Wired

### Camera make and model

N/A

**Comment by bchhabra** [comment]
I am facing the excatly the same issue. Only Online message are being sent and no Offline.
@blakeblackshear  could you please check this?
